PendingReleaseNotes

```diff
@@ -0,0 +1,9 @@
+v10.2.0
```

Comment:
at this point you can modify doc/release-notes.rst directly for what's currently the jewel rc section